src/qtouch/qtouch.c

```diff
@@ -524,7 +536,13 @@ void qtouch_timer_config(void)
     Timer_task.cb = qtouch_timer_task_cb;
     Timer_task.mode = TIMER_TASK_REPEAT;
 
+    static uint8_t timer_task_added = 0;
+    if (timer_task_added) {
+        timer_remove_task(&TIMER_0, &Timer_task);
```

Comment:
It isn't really needed because `timer_add_task` won't add if it alreayd exists in the list. Not sure why they added it. Maybe compatibility with other chips.